tests/framework/installer/cassandra_manifests.go

```diff
@@ -0,0 +1,279 @@
+package installer
+
+import (
+	"fmt"
+	cassandrav1alpha1 "github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1"
+)
+
+type CassandraManifests struct{}
+
+func (i *CassandraManifests) GetCassandraCRDs() string {
+	return `apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusters.cassandra.rook.io
+spec:
+  group: cassandra.rook.io
+  names:
+    kind: Cluster
+    listKind: ClusterList
+    plural: clusters
+    singular: cluster
+  scope: Namespaced
+  version: v1alpha1`
+}
+
+func (i *CassandraManifests) GetCassandraOperator(namespace string) string {
+
+	return fmt.Sprintf(`
+# Namespace where Cassandra Operator will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: %[1]s
+
+---
+# ClusterRole for cassandra-operator.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: rook-cassandra-operator
+rules:
+  - apiGroups:
+    - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - persistentvolumes
+      - persistentvolumeclaims
+    verbs:
+      - get
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - roles
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
```

Comment:
wanted to make sure you saw the follow-up discussion about this at https://github.com/rook/rook/pull/2207#discussion_r237352283, which has been marked by github as outdated

Reply:
Yeap ! 
I followed the model of rook, where all rbac permissions are created along with the cluster instance. 
Now the operator doesn't have permissions on serviceaccounts, roles and rolebindings.

Reply:
So this granting of permissions to the `rbac.authorization.k8s.io` API group can be removed from this ClusterRole definition, right? I would like us to remove if since it sounds like we do not need it.